### ISSUE TITLE
IdsIcon: Should allow transparent background

### ISSUE THREAD

**Issue by mecharmor** [infor-design/enterprise-wc#1790]
**Is your feature request related to a problem or use case?**
The ids-icon has many amazing themes to switch but it seems there are certain aspects of the theme I cannot hide/disable if needed. For example if I just want to show the icon and have the icon color a certain way then i'm forced to keep a circle background that is pulled from the ids theme.
See:
<img width="114" alt="image" src="https://github.com/infor-design/enterprise-wc/assets/12702175/9cc27f80-de8c-46b9-a8e9-0ce3dcc6ca2c">

**Describe the solution you'd like**
I would like some way to remove the background from the icon with a theme applied.
Expected usage:
```
<ids-icon size="xl" icon="calculator" status-color="azure" background="transparent"></ids-icon>
```

**Describe alternatives you've considered**
N/A - we are locked into the attributes IDS provides and right now there is not a clean way to handle this

**Additional context**
Basically we need a way to show the icon without anything else.

I also found that status-color `slate` does not exist. Could we add that in this change?

cc: @tmcconechy

**Comment by mecharmor** [comment]
@tmcconechy we also might be mixing implementations with ids-icon. for example you may have `<ids-icon` and `<ids-badge`. This way we don't need to disable certain feature sets.

**Comment by tmcconechy** [comment]
@mecharmor 

If you dont add status-color. You dont get those status colors backgrounds. https://main.wc.design.infor.com/ids-icon/status-color.html and by default it doesnt have a background. 

But for fill/color i see a bug. There is a setting `fill="#800"` but it should be color because svg uses the color. So the only way would be to override the css variable which you can do. I broke it when adding these vars.

``` 
#my-icon {
--ids-icon-color-default: red;
}
```
 
I'll fix that on here but also we are changing to an svg font #1191 so doesnt change much just the setting should maybe be color.

**Comment by mecharmor** [comment]
Do you know if there is anyway for me to extract the raw SVG from the IDS component so I can color it externally?

**Comment by tmcconechy** [comment]
@mecharmor what is it exactly you need to do? Maybe we can make it easier. But i think the css variable is the best way if its just setting the color.

If you really really just want the svg insides you could use this JSON file https://github.com/infor-design/enterprise-wc/blob/main/src/components/ids-icon/ids-icon.ts#L1 

But it will be going away when i finish #1191 where it will use an icon font and css. So not sure its a good idea.

**Comment by mecharmor** [comment]
I am trying to embed an ids-icon inside a cytoscape node label via html. But when I try to assign a certain color scheme to the icon I get the background shown in image in the description above. Here is the lib for reference: https://github.com/kaluginserg/cytoscape-node-html-label

I may avoid the CSS variable approach and wait until it becomes a blocker issue just to avoid scenarios where i'm stomping a bug that I can just negate by upgrading soon. I see issue #1191 is set for Feb sprint so that should align with our current roadmap just fine.

**Comment by mecharmor** [comment]
Thank you @tmcconechy . I tried playing around with color/fill but I couldn't seem to get it to work. When I looked at the documentation I found that "Color" was listed as an attribute and "fill" was an attribute when I dug into the code but I am not sure what values they accept.
https://github.com/infor-design/enterprise-wc/blob/main/src/components/ids-icon/README.md

**Comment by tmcconechy** [comment]
Fixing this on the PR being applied shortly. To clarify i just fixed `color="red-10"` The colors are also all changed so you should be able to do it this way in beta.19

```
<ids-icon size="xl" icon="calculator" color="azure-60" background="transparent"></ids-icon>
```

**Comment by jbrcna** [comment]
QA Passed
<img width="865" alt="image" src="https://github.com/infor-design/enterprise-wc/assets/56722184/43972912-811d-4e5d-9fa2-f0d4c70cc32f">
